client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -39,6 +40,7 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
       step = 1,
       multiplier = 10,
       value,
+      noFormatting = false,
```

Comment:
adding this - without it the port number is reformatted at `9,100` which isn't what I want at all